packages/dredd-transactions/scripts/postpack.js

```diff
@@ -13,7 +13,12 @@ if (fs.existsSync(SYMLINKS_LOG)) {
     .split('\n')
     .filter(line => line.trim())
     .map(dependencyName => path.join(PACKAGE_DIR, 'node_modules', dependencyName))
-    .filter(symlinkPath => fs.existsSync(symlinkPath))
+    .map((a) => console.log(a) || a)
+    .filter(symlinkPath => {
+      const exists = fs.existsSync(symlinkPath)
+      console.log('checking if %s exists', symlinkPath, exists)
```

Comment:
The issue is that symlink to a scoped package (i.e. `@apielements/apib-parser`) never resolves, even if it exists.